plugins/modules/web_infrastructure/jenkins_job_info.py

```diff
@@ -33,12 +33,10 @@
     type: str
     description:
       - Password to authenticate with the Jenkins server.
-      - This is a required parameter, if C(token) is not provided.
```

Comment:
Can you replace this with `This is mutually exclusive with C(token).`? (That should have already been mentioned there before IMO.) Same for the other.

Reply:
Done in the latest commit.